### PR TITLE
test: s/osd_objectstore_type/osd_objectstore

### PR DESCRIPTION
**Pull request by xiexingguo**
Signed-off-by: xie xingguo <xie.xingguo@zte.com.cn>

### CODE REVIEW

**Comment by xiexingguo** [comment]
retest this please

File: qa/workunits/cephtool/test.sh [hunk 1]
```diff
@@ -1678,7 +1678,7 @@ function test_mon_osd_pool()
   ceph osd pool create ec_test 1 1 erasure
   ceph osd pool application enable ec_test rados
   set +e
-  ceph osd metadata | grep osd_objectstore_type | grep -qc bluestore
+  ceph osd metadata | grep osd_objectstore | grep -qc bluestore
```
Comment (tchaikov) [inline qa/workunits/cephtool/test.sh]: better off switching to `jq`.

**Comment by xiexingguo** [comment]
switch to "ceph osd count-metadata", now the output is simple enough and let's not bother using jq here.

**Comment by xiexingguo** [comment]
seems the original test-case is broken and never functionable, repushed

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-07-24_03:44:49-rados-wip-sage-testing-distro-basic-smithi/

**Comment by liewegas** [comment]
cephtool is failing for other reasons, not merging this just yet.

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-07-25_20:13:42-rados-wip-sage-testing-distro-basic-smithi/

**Pull request merged**